app/models/card.rb

```diff
@@ -13,7 +13,7 @@ class Card < ActiveRecord::Base
   validates :deck_name, presence: true, if: :deck_name
   validate :words_cannot_be_equal
 
-  scope :condition,         -> { where('review_date <= ?', Time.now) }
+  scope :condition,         -> { where('review_date <= ?', Time.zone.now) }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.